helm/alfresco-content-services/charts/alfresco-search/templates/deployment.yaml

```diff
@@ -76,7 +76,7 @@ spec:
               exec:
                 command: ["/bin/bash", "-c", "sleep 10"]
       initContainers:
-        - name: init-db
+        - name: init-fs
```

Comment:
dont we need the if clause here also?, we have the value below but we are not using it :)

Reply:
No, because as I found out with @alxgomz alfresco-search is using static gp2 volume, so no need for disabling chown here.